demo-shell/src/app/components/files/files.component.ts

```diff
@@ -53,7 +54,10 @@ const DEFAULT_FOLDER_TO_SHOW = '-my-';
 @Component({
     selector: 'app-files-component',
     templateUrl: './files.component.html',
-    styleUrls: ['./files.component.scss']
+    styleUrls: ['./files.component.scss'],
```

Comment:
@DenysVuika I had to add this provider to this component to make it work... This component is using the adf-start-form component and without this the widgets coming from ProcessFormRenderingService would not work, I don't think it's the correct implementation but I don't know what else we could do here.

I noticed that we're also injecting this provider in the Form Component in the demoshell for the same reason.

Any idea?